.circleci/config.yml

```diff
@@ -80,7 +80,8 @@ jobs:
                       - v0.4-{{ checksum "setup.py" }}
             - run: sudo apt-get -y update && sudo apt-get install -y libsndfile1-dev
             - run: pip install --upgrade pip
-            - run: pip install .[sklearn,tf-cpu,torch,testing,sentencepiece,speech,vision]
+            - run: pip install .[sklearn,tf-cpu,testing,sentencepiece,speech,vision]
+            - run: pip install -U torch==1.8.1 torchvision==0.9.1 torchaudio==0.8.1
```

Comment:
I don't think this will work, shouldn't it be:

```
pip install torch==1.8.1+cu111 torchvision==0.9.1+cu111 torchaudio==0.8.1 -f https://download.pytorch.org/whl/torch_stable.html
```

not sure which cuda we are on

Reply:
CircleCI does not run GPU tests. The Github Actions should be updated accordingly.

Reply:
CI seems to be ok, how does it know which cuda version to pick.

Reply:
ah, of course ;) All is good then.

Reply:
Actually the Github Actions tests are on a fixed docker which is pinned to 1.8.0, so there's no need to update these as long as we don't upgrade to 1.9.0.